src/java.base/share/classes/jdk/internal/loader/ClassLoaders.java

```diff
@@ -55,15 +55,23 @@ private ClassLoaders() { }
     private static final PlatformClassLoader PLATFORM_LOADER;
     private static final AppClassLoader APP_LOADER;
 
+    // Sets the ServicesCatalog for the specified loader using archived objects.
+    private static void setArchivedServicesCatalog(
+            ArchivedClassLoaders archivedClassLoaders, ClassLoader loader)
```

Comment:
Thanks for the update. The formatting on L60 is messed now, can't the ArchivedClassLoaders go on the previous line? I don't have any other comments, it's worked out well.

Reply:
How about the latest version (4cd981c)? I removed the `archivedClassLoaders` argument; this also makes the callers simpler.

Reply:
Looks good to me.

Reply:
This makes it simpler and much nicer - thank you!